pyproject.toml

```diff
@@ -124,6 +127,6 @@ ignored-classes = [
 ]
 
 [tool.mypy]
-python_version = 3.12
+python_version = 3.10
```

Comment:
Is this change on purpose?

Reply:
Yes, this was wrong before. Mypy should check against the oldest supported version.